Jellyfin.Server/Migrations/JellyfinMigrationService.cs

```diff
@@ -165,7 +169,8 @@ public async Task MigrateStepAsync(JellyfinMigrationStageTypes stage, IServicePr
                 }
                 catch (Exception ex)
                 {
-                    logger.LogCritical(ex, "Migration {Name} failed", item.Key);
+                    logger.LogError(ex, "Migration {Name} failed", item.Key);
```

Comment:
Is it intentional to downgrade the _normal_ `logger` to `LogError(` from `LogCritical(`? Seems an odd change.

Reply:
Yes, Critical errors are already logged with higher verbosity though the startuplogger so no need to log that twice